### PR TITLE
DOC: Updated Troubleshooting Guide

### PR DESCRIPTION
**Pull request by Abhinav-26**
# Description

updated few more troubleshooting steps in Devtron's troubleshooting guide.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update

# Checklist:

* [X] The title of the PR states what changed and the related issues number (used for the release note).
* [X] Does this PR require documentation updates?
* [X] I've updated documentation as required by this PR.
* [ ] I have performed a self-review of my own code
* [ ] I have commented my code, particularly in hard-to-understand areas
* [ ] I have tested it for all user roles

### CODE REVIEW

File: docs/FAQs/devtron-troubleshoot.md [hunk 1]
```diff
@@ -96,4 +96,28 @@ https://grafana.com/api/dashboards/13320/revisions/4/download
 https://grafana.com/api/dashboards/13325/revisions/4/download
 https://grafana.com/api/dashboards/13321/revisions/6/download
 ```
-After that, your issue should be resolved and you should be able to see all the graphs on UI.
\ No newline at end of file
+After that, your issue should be resolved and you should be able to see all the graphs on UI.
+
+#### 6. Unable to Login Devtron dashboard even if the password is correct
+
+If you are not able to login your Devtron dashboard after giving the correct password, there can be high chance that the `argocd token` of previous session has been stored in the cookies and is not able to override the new token that is generated for the new session. If this is the issue you are facing, please follow the following steps.
```
Comment (pghildiyal) [inline docs/FAQs/devtron-troubleshoot.md:103]: solve for chrome, firefox, safari

Comment (Abhinav-26) [reply docs/FAQs/devtron-troubleshoot.md:103]: solved for all three.

File: docs/FAQs/devtron-troubleshoot.md [hunk 2]
```diff
@@ -96,4 +96,55 @@ https://grafana.com/api/dashboards/13320/revisions/4/download
...
```
Comment (pghildiyal) [inline docs/FAQs/devtron-troubleshoot.md:103]: If you are not able to login into Devtron dashboard even after giving the correct password, it is possible that the `argocd token` of previous session has been stored in the cookies and is not able to override the new token that is generated for the new session.

File: docs/FAQs/devtron-troubleshoot.md [hunk 3]
```diff
@@ -96,4 +96,55 @@ https://grafana.com/api/dashboards/13320/revisions/4/download
...
+In the Devtron's Discover Chart section, if you are not able to see any charts available, goto `Global Configuration` >> `Chart Repositories` and click on `Refresh Chart` at the top-right as shown in the image below. After clicking the button, it might take 4-5mins to show all the charts in `Discover` section depending upon the chart repositories added. 
+![charts-not-found](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/devtron-troubleshooting/refresh-charts.png)
+
+
+#### 8. Not able to update cluster
+
+In `Global Configurations` >> `Cluters & Environments`, if you try to update a cluster which has been already added in Devtron, you might get an error as `{"message":"Failed to update datasource. Reload new version and try again"}`. If you are facing such issue, please follow the following steps - 
+
+1. Edit the changes you want to make in respective cluster
+2. Click on save after making changes, hopefully you will get error message as stated above.
```
Comment (pghildiyal) [inline docs/FAQs/devtron-troubleshoot.md:141]: Click on save after making changes and you may get error message stated above.

File: docs/FAQs/devtron-troubleshoot.md [hunk 4]
```diff
@@ -96,4 +96,55 @@ https://grafana.com/api/dashboards/13320/revisions/4/download
...
+3. Go to cluster where devtron has been installed and execute - `kubectl -ndevtroncd delete po -l app=devtron`
+4. Now refresh the page and you should be able to save it.
+
+[Note: If you already have created some environments in that cluster, its needs to be updated again, `click on environment` >> `click on update`]
```
Comment (pghildiyal) [inline docs/FAQs/devtron-troubleshoot.md:145]: Note: If you already have created some environments in that cluster, it needs to be updated again

File: docs/FAQs/devtron-troubleshoot.md (hunk 2 above)
Comment (Abhinav-26) [reply docs/FAQs/devtron-troubleshoot.md]: resolved

File: docs/FAQs/devtron-troubleshoot.md (hunk 3 above)
Comment (Abhinav-26) [reply docs/FAQs/devtron-troubleshoot.md]: resolved

File: docs/FAQs/devtron-troubleshoot.md (hunk 4 above)
Comment (Abhinav-26) [reply docs/FAQs/devtron-troubleshoot.md]: resolved

**Review by pghildiyal — APPROVED** [review]

**Pull request merged**